repository/build.gradle

```diff
@@ -6,6 +6,7 @@ dependencies {
     implementation group: 'org.springframework', name: 'spring-jdbc', version: springVersion
     implementation group: 'org.springframework', name: 'spring-context', version: springVersion
     implementation group: 'org.springframework', name: 'spring-beans', version: springVersion

```

Comment:
Please use SpringBoot starters

Reply:
As I know, I'm focusing in this PR at JPA integration, next PR - Spring Boot migration